src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx

```diff
@@ -25,15 +28,16 @@ export const AboutPartner: React.FC<AboutPartnerProps> = ({
 
       <Column span={6}>
         {profile?.fullBio && <Text variant="text">{profile.fullBio}</Text>}
-        {website && (
+        {canRenderWebsite && (
+          /* @ts-expect-error STRICT_NULL_CHECK */
```

Comment:
Shouldn't be introducing new ignores here — I don't see why the guard he wouldn't be sufficient though.

Reply:
![Screen Shot 2021-05-14 at 5 27 30 PM](https://user-images.githubusercontent.com/79980131/118284952-ac817400-b4d9-11eb-879a-0b80418af004.png)

Reply:
If we know for sure it'll never be null I'd rather use an assertion `{website!}` vs the ts-ignore

Reply:
Fixed